static/app/components/acl/featureDisabled.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import {Fragment} from 'react';
+
+import FeatureDisabled from 'sentry/components/acl/featureDisabled';
+import storyBook from 'sentry/stories/storyBook';
+
+export default storyBook('FeatureDisabled', story => {
+  story('Default', () => {
+    return (
+      <Fragment>
+        <p>
+          The FeatureDisabled component is used to display a warning message when a user
+          attempts to access a feature that is not available to them. It can show both a
+          simple alert-style message as well as an expandable help section with more
+          detailed information. The component supports customization of the message,
+          feature name, and whether to show the help toggle. It's commonly used throughout
+          the application to gracefully handle feature access restrictions and provide
+          clear feedback to users about why certain functionality may be unavailable.
+        </p>
+        <FeatureDisabled
+          features={['this-feature-does-not-exist']}
+          featureName="Feature"
+        />
```

Comment:
Want to do the full version with description etc

Reply:
Also seems unrealted to button label?

Reply:
I added it because I didn't want to track this component somewhere in the UI, and adding a story felt like a simple win :D